2024-11-26 13:10:03,395 - main.py[line:44] - INFO: 已删除文件：data_visualization\all_words.csv
2024-11-26 13:10:03,402 - main.py[line:44] - INFO: 已删除文件：data_visualization\predicted_results.csv
2024-11-26 13:10:03,403 - main.py[line:44] - INFO: 已删除文件：data_visualization\positive_words.csv
2024-11-26 13:10:03,404 - main.py[line:44] - INFO: 已删除文件：data_visualization\negative_words.csv
2024-11-26 13:10:03,405 - main.py[line:44] - INFO: 已删除文件：model\processed_data.csv
2024-11-26 13:10:03,406 - main.py[line:44] - INFO: 已删除文件：weibo_crawler\weibo_data.csv
2024-11-26 13:10:03,407 - main.py[line:44] - INFO: 已删除文件：static\wordclouds\pie_chart.png
2024-11-26 13:10:03,410 - main.py[line:44] - INFO: 已删除文件：static\wordclouds\wordcloud_all.png
2024-11-26 13:10:03,412 - main.py[line:44] - INFO: 已删除文件：static\wordclouds\wordcloud_positive.png
2024-11-26 13:10:03,413 - main.py[line:44] - INFO: 已删除文件：static\wordclouds\wordcloud_negative.png
2024-11-26 13:10:05,211 - main.py[line:24] - ERROR: Error executing Scrapy command: Command '['scrapy', 'crawl', 'search']' returned non-zero exit status 1.
2024-11-26 13:10:05,213 - text_processor.py[line:20] - ERROR: CSV File ../weibo_crawler/weibo_data.csv not found.
2024-11-26 13:10:05,218 - __init__.py[line:113] - DEBUG: Building prefix dict from the default dictionary ...
2024-11-26 13:10:05,219 - __init__.py[line:132] - DEBUG: Loading model from cache C:\Users\DoubleT\AppData\Local\Temp\jieba.cache
2024-11-26 13:10:06,193 - __init__.py[line:164] - DEBUG: Loading model cost 0.974 seconds.
2024-11-26 13:10:06,194 - __init__.py[line:166] - DEBUG: Prefix dict has been built successfully.
2024-11-26 13:11:08,062 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\all_words.csv
2024-11-26 13:11:08,063 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\predicted_results.csv
2024-11-26 13:11:08,063 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\positive_words.csv
2024-11-26 13:11:08,064 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\negative_words.csv
2024-11-26 13:11:08,064 - main.py[line:36] - ERROR: 文件路径不存在：model\processed_data.csv
2024-11-26 13:11:08,065 - main.py[line:36] - ERROR: 文件路径不存在：weibo_crawler\weibo_data.csv
2024-11-26 13:11:08,065 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\pie_chart.png
2024-11-26 13:11:08,066 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_all.png
2024-11-26 13:11:08,066 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_positive.png
2024-11-26 13:11:08,066 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_negative.png
